go/libkb/gpg_cli.go

```diff
@@ -315,66 +299,9 @@ func (g *GpgCLI) MakeCmd(args []string) *exec.Cmd {
 	} else {
 		nargs = args
 	}
+	if g.G().Service {
```

Comment:
Without this flag, GPG will try to open `/dev/tty` and if it fails, will just crash out.